lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -2921,11 +2922,11 @@ void SpecialConstantOp::getAsmResultNames(OpAsmSetValueNameFn setNameFn) {
   specialName << 'c';
   specialName << static_cast<unsigned>(getValue());
   auto type = getType();
-  if (type.isa<ClockType>()) {
+  if (firrtl::type_isa<ClockType>(type)) {
```

Comment:
Maybe the alias name should be used / part of the result name instead of stripped away?